types/gensync/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "space-before-function-paren": false
```

Comment:
is this one required? It can be a prettier nitpit setting, that can be disabled via prettier inline comment

Reply:
Required, no, but every single use of a generator (so, every single test) would need to have a prettier comment.

Reply:
I'm somewhat inclined to go figure out if there's something in dtslint that can be changed to not complain about this particular code. But, then again, I would really like DT to enforce prettier-d code in the first place, so what can I do? 😄

Reply:
oh, really sorry to miss a point here. I"m pretty sure Prettier always space this:
https://github.com/prettier/prettier/issues/3845#issuecomment-362211847
There is no way to add a configuration and I'd avoid always adding custom exclusions to keep things simple. I'd modify default global settings used here via inclusion:
https://github.com/microsoft/DefinitelyTyped-tools/blob/master/packages/dtslint/dt.json
and add this exclusion, as it cannot be (and won't be) configured via Prettier config

Reply:
All good. Prettier wants `function* () {}`, dtslint (apparently) wants `function*()`. Will look to see what I can do about that.

Reply:
I'd really opt to add this change upstream, I had the same problem couple of times here. This will eventually incrementally increase the toil on developers, if not corrected.